smart_pet_buddy/pubspec.lock

```diff
@@ -406,6 +406,27 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.1.0"
+  video_player:
```

Comment:
Nice, does this allow us to have a video on the homepage?

Reply:
Nope, I tried to but gave up and just added a picture instead😅 I'll remove that part